sources/core/tests/saveable_operators_tests.cpp

```diff
@@ -93,3 +93,29 @@ TEST(SaveableOperators, VectorOperators)
     EXPECT_EQ(other_vector[5], 43);
     EXPECT_EQ(Hash(other_vector), Hash(vector));
 }
+
+TEST(SaveableOperators, bitset)
+{
+    kv::FastSerializer serializer;
+    for(int i = 0; i < 16; i++)
+    {
+        serializer << std::bitset<16>(i);
```

Comment:
The `16` in the `std::bitset` template parameter is an amount of bits, not the maximum possible value or something.
So here `std::bitset<4>` should be enough.

Reply:
Oops, I usually don't confuse